routes/wishlist.routes.js

```diff
@@ -29,5 +29,10 @@ router.delete(
   passport.authenticate('jwt', { session: false }),
   wishlistController.deleteWishlistItem,
 );
+router.get(
+  '/sorting/:page/:field/:order',
```

Comment:
I would suggest to pass this params as query params, for example URL should looks like /sorting?page=1&field=name&order=asc